examples/authorization_details.json

```diff
@@ -1,12 +1,6 @@
 [
    {
       "type": "openid_credential",
-      "format": "jwt_vc_json",
-      "credential_definition": {
-         "type": [
-            "VerifiableCredential",
-            "UniversityDegreeCredential"
-         ]
-      }
+      "credential_description": "UniversityDegreeCredential"
```

Comment:
@paulbastian I'm wndering if the lookup could be based on a parameter called "description"

what about `credential_id` or something different than `_description`?

Reply:
Agreed if we are intending to use this as a look up or reference description feels like the wrong term here.

Reply:
Discussion in editors call:
As the parameter in credential offer is called `credentials`, name this one equivalently?
Probably this parameter is a misnomer, but renaming this would be a bigger breaking change

Reply:
On today's editors call we discussed that this should be consistent with `credentials_supported` in the issuer metadata, and `credentials` in the credential offer, which would imply the name should be `credential`.

The alternative would be to rename all these parameters, which not everyone on the call was keen on, but would be best discussed under https://github.com/openid/OpenID4VCI/issues/91

Reply:
@peppelinux @Sakurann @jogu @tplooker please review